modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -25,62 +23,75 @@ Carefully review the following consequences before opting in to migration:
+//
+//==== Prerequisites
 
-===== Procedure
+//* Access to the {product-title} web console.
 
-To opt in to automatic CSI migration for vSphere:
+//* Access to the cluster with cluster-admin privileges.
 
-. Log in to the web console.
+//===== Procedure
 
-. Click *Administration* -> *CustomResourceDefinitions*.
+//To opt in to automatic CSI migration for vSphere:
 
-. On the *CustomResourceDefinitions* page, type "Storage" to find the `Storage` custom resource (CR).
+//. Log in to the web console.
 
-. Click the *Storage* CR.
+//. Click *Administration* -> *CustomResourceDefinitions*.
 
-. On the *storages.operator.openshift.io* page, click the *Instances* tab.
+//. On the *CustomResourceDefinitions* page, type "Storage" to find the `Storage` custom resource (CR).
 
-. Click the name of the desired instance, and then click the *YAML* tab.
+//. Click the *Storage* CR.
 
-. Set the `spec.vsphereStorageDriver` parameter to `CSIWithMigrationDriver`, as shown in the following example:
-+
-[source, yaml]
-----
-....
-spec:
-  logLevel: Normal
-  managementState: Managed
-  operatorLogLevel: Normal
-  vsphereStorageDriver: CSIWithMigrationDriver <1>
-...
-----
-<1> `spec.vsphereStorageDriver` parameter set to `CSIWithMigrationDriver`
+//. On the *storages.operator.openshift.io* page, click the *Instances* tab.
 
-. Click *Save*.
+//. Click the name of the desired instance, and then click the *YAML* tab.
 
-=== Using the CLI to opt in to automatic CSI migration
+//. Set the `spec.vsphereStorageDriver` parameter to `CSIWithMigrationDriver`, as shown in the following example:
+//+
+//[source, yaml]
+//----
+//....
+//spec:
+//  logLevel: Normal
+//  managementState: Managed
+//  operatorLogLevel: Normal
+//  vsphereStorageDriver: CSIWithMigrationDriver <1>
+//...
+//----
+//<1> `spec.vsphereStorageDriver` parameter set to `CSIWithMigrationDriver`
 
-==== Prerequisites
+//. Click *Save*.
 
-* Access to the cluster with cluster-admin privileges.
+//=== Using the CLI to opt in to automatic CSI migration
 
-==== Procedure
+//==== Prerequisites
 
-To opt in to automatic CSI migration for vSphere, run the following command:
+//* Access to the cluster with cluster-admin privileges.
 
-[source, cli]
-----
-oc patch storage cluster --type=merge -p '{"spec":{"vsphereStorageDriver":"CSIWithMigrationDriver"}}'
-----
+//==== Procedure
 
-[NOTE]
-====
-You can determine when migration is complete by verifying that the `VSphereMigrationControllerAvailable` condition is set to "true" in the `Storage` object.
-====
+//To opt in to automatic CSI migration for vSphere, run the following command:
+
+//[source, cli]
+//----
+//oc patch storage cluster --type=merge -p '{"spec":{"vsphereStorageDriver":"CSIWithMigrationDriver"}}'
+//----
+
+//[NOTE]
+//====
+//You can determine when migration is complete by verifying that the `VSphereMigrationControllerAvailable` condition is set to "true" in the `Storage` object.
+//====
```

Comment:
All of this needs to be removed.

Reply:
yes all the opt-in 4.13 content needs to be removed

Reply:
Done.

Reply:
Outdated. Fixed.